dpp/src/identity/script.rs

```diff
@@ -0,0 +1,73 @@
+use core::fmt;
+
+use dashcore::Script as DashcoreScript;
+use serde::{Deserialize, Serialize};
+
+use crate::{
+    util::string_encoding::{self, Encoding},
+    ProtocolError,
+};
+
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub struct Script(DashcoreScript);
+
+impl Script {
+    pub fn is_p2sh(&self) -> bool {
```

Comment:
Shall we implement Deref pattern so all Script methods will "accessible" with CoreScript as well?